packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -228,6 +233,9 @@ class TransitLegBody extends Component<Props, State> {
               </S.LegClickableButton>
             </S.LegDescription>
           </S.LegClickable>
+          {RouteDescriptionFooter && trimetMode && (
```

Comment:
Defining `RouteDescriptionFooter` redundant with setting `trimetMode` to `true`. What do you think @miles-grant-ibigroup?

Reply:
What should I do there instead?

Reply:
I agree! I like this approach